swiggyclone/service/serializers.py

```diff
@@ -68,49 +59,32 @@ def create(self, validated_data):
 
 
 class BranchSerializer(serializers.ModelSerializer):
-    foodItems = FoodItemSerializer(many=True)
-    orders = OrderSerializers(many=True)
-    id = serializers.IntegerField(required=False)
+    
+   
     class Meta:
         model = Branch
         fields = (
                   'id',
                   'name',
                   'address',
-                  'pincode',
-                  'foodItems',
-                  'orders',
-                  'restaurent_id',
-                  'branchOwner_id'
+                  'pincode',    
+                  'branchOwner',
+                  'restaurent'
                 )
-    def create(self, validated_data):
-        foodItems_data = validated_data.pop('foodItems')
-        orders_data = validated_data.pop('orders')
-        branch = Branch.objects.create(**validated_data)
-        for foodItem_data in foodItems_data:
-            FoodItem.objects.create(branch=branch, **foodItem_data)
-        for order_data in orders_data:
-            Order.objects.create(branch=branch, ** order_data)
-        return branch
+    
 
 
 class RestaurentSerializer(serializers.ModelSerializer):
-    branches = BranchSerializer(many=True)
-    id = serializers.IntegerField(required=False)
+    
+   
     class Meta:
         model = Restaurent
         fields = (
                   'id',
                   'name',
                   'discription',
-                  'branches',
                 )
-    def create(self, validated_data):
-        branches_data = validated_data.pop('branches')
-        restaurent = Restaurent.objects.create(**validated_data)
-        for branch_data in branches_data:
-            Branch.objects.create(restaurent=restaurent, **branch_data)
-        return restaurent
+    
 
 
 
```

Comment:
remove the unnecessary lines at the end of files. there should be one empty line at the end of the file. not less or more